### ISSUE TITLE
Can't get to asteroid

### ISSUE THREAD

**Issue by NateDaBeast** [MuMech/MechJeb2#664]
Okay so this is what I'm doing.

Going to launch pad
Using Ascent Guidance on MechJeb to get myself at 100km
Maneuver Planner I use "Hohmann transfer to target" after selecting the asteroid as target
Maneuver Planner then says "target for Hohmann transfer must be in the same sphere of influence"

It also sometimes says something about the hyperbolic curve or something
Example - http://i.imgur.com/LDfxnob.png

So what exactly do I need to do to get near or try to land on the asteroid using mechjeb ?

Thanks for reading!

**Comment by robot256** [comment]
Hi Nate,
I saw your forum thread with the same question and hope the answers there were useful. (http://forum.kerbalspaceprogram.com/index.php?/topic/119779-cant-get-to-asteroid-noob-amp-using-mechjeb/)  The short answer, of course, is "exactly what you have to do if you are not using Mechjeb".  Mechjeb does not have superpowers, it merely does calculations and actions for you that you could otherwise do by hand.

As the members there stated, Mechjeb is operating as intended.  Hohmann transfers only work when both you and your target are in relatively circular orbits around the same moon, planet or star.  Thus, if the asteroid is around the Sun and you are orbiting Kerbin, it won't work, and if the asteroid is on a flyby trajectory near Kerbin it's not circular and won't work.  You have to go orbit the Sun first.

The only asteroid missions I have done, I grabbed them in solar orbit by plotting the maneuvers manually.  I really like using the "+/-" buttons in the Mechjeb Maneuver Editor, and having Mechjeb execute the maneuvers.  But it still takes practice to set the intercept point just right.

You might try one of the newer Mechjeb features, "Intercept Target at Specific Time".  This might be able to plot an intercept course without relying on a Hohmann Transfer orbit.  But you also run the risk of using way too much fuel and not being able to slow down to meet it, so you have to be very careful what parameters you put in.

Now I'm just speculating, but you might also be able to use the "Depart for Planet" function to head towards a sun-orbiting asteroid from Kerbin orbit.  That is technically the maneuver you would want to use, but I have no idea if Mechjeb will do it properly for an asteroid that is really close to Kerbin.

In short, the only way to grab an asteroid is to understand the series of maneuvers required, and then program them into Mechjeb at the right times.  Any asteroid tutorial, Mechjeb or no, will show you the right sequence of maneuvers, then Mechjeb will simply make plotting and executing them easier than it would be by hand.  Good luck!

I will suggest that this issue ticket be closed, as it does not reference a specific bug or feature of the Mechjeb code.
Tom
